test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1284,9 +1289,9 @@ class test_ucp_sockaddr_wireup_fail_try_next_cm :
         return 0;
     }
 
-    virtual bool wait(entity &e, uint32_t wait_ep_flags)
+    virtual bool test_all_ep_flags(entity &e, uint32_t flags)
     {
-        if (test_ucp_sockaddr_wireup_fail::wait(e, wait_ep_flags)) {
+        if (test_ucp_sockaddr_wireup_fail::test_all_ep_flags(e, flags)) {
```

Comment:
it is needed to use method of base class

Reply:
should not derived class call overridden method by default?

Reply:
but we need to call base method instead of derived one

Reply:
> i see, you are right, but the implementation looks weird:
> 
> 1. this func is used in the loop, so if base impl returns true (some flag was not set), we a skipping the test.

yes, if flag `UCP_EP_FLAG_CLIENT_CONNECT_CB` is set, then no need to run the test, since trying the next CM won't be done.
the idea of the test is to ensure that trying of the next CM is scheduled and emulate EP failure.

> 2. i guess skip message is based on some special flag?

`UCP_EP_FLAG_CLIENT_CONNECT_CB` is a flag which is checked by the test.